### PR TITLE
#166236243 Deploy app to heroku

### PR DESCRIPTION
**Pull request by Tyak99**
### What does this PR do?

* Deploy the app to heroku
* Set up Continous Delivery on heroku to build pull requests

### Description of Task to be completed?

* Add a server folder to contain project folders
* Add build script to package json
* Add scripts to build heroku

### Any background context you want to provide?

N/A

### What are the relevant Github issues?

N/A

### Questions:

N/A

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,14 +4,23 @@
...
-    "test": "npm run test:migrate && NODE_ENV=test nyc --reporter=html --reporter=text mocha --require @babel/register --require babel-polyfill ./test/ --recursive --exit",
+    "test": "npm run test:migrate && NODE_ENV=test nyc --reporter=html --reporter=text mocha --require @babel/register --require babel-polyfill ./server/test/ --recursive --exit",
     "migrate": "sequelize db:migrate",
     "undo-all:migrate": "sequelize db:migrate:undo:all",
+    "refreshdb": "npm run undo-all:migrate && npm run migrate",
     "undo-current:migrate": "sequelize db:migrate:undo",
     "test:migrate": "NODE_ENV=test npm run undo-all:migrate && NODE_ENV=test npm run migrate ",
-    "start": "babel-node ./index.js",
+    "start": "node --require babel-polyfill ./build/index.js",
+    "start:dev": "nodemon --exec babel-node ./server/index.js",
```
Comment (waleCloud) [inline package.json:14]: Hi @Tyak99 kindly share some light as to why you're using babel-node?

**Comment by waleCloud** [comment]
Hey man @Tyak99 Good job.
you have 9 commit messages on this PR, is there a particular reason?

File: package.json (hunk 1 above)
Comment (Tyak99) [reply package.json:14]: @waleCloud we are using babel-node on development to transpile the new es6 syntax. syntax like `import module from 'moduleName'`

**Comment by Tyak99** [comment]
> Hey man @Tyak99 Good job.
> you have 9 commit messages on this PR, is there a particular reason?

@waleCloud Yes, I had to push some new commits after creating the PR, so as to meet the criteria of the task I was given

**Review by fortunee — APPROVED** [review]
LGTM

**Pull request merged**